constellations/orion/src/tests_integration/semester_integration_test.go

```diff
@@ -13,9 +13,9 @@ import (
 
 // Test: Create 3 Semesters and GetAll(false)
 func Test_CreateSemesters(t *testing.T) {
-	semester1 := createSemester("2020_spring", "Spring 2020")
-	semester2 := createSemester("2020_fall", "Fall 2020")
-	semester3 := createSemester("2020_winter", "Winter 2020")
+	semester1 := createSemester("2020_spring", "Spring 2020", 3)
+	semester2 := createSemester("2020_fall", "Fall 2020", 2)
+	semester3 := createSemester("2020_winter", "Winter 2020", 1)
```

Comment:
can you rename these to actually reflect what might happen in a school year.
i.e. 2020_spring comes first? then 2020_fall, then 2020_winter.
But we want to test if ordering works, so you'll have to rename one of them to 2019 or 2021 for example.